File: core/dao/models/Group.py
# -*- coding: utf-8 -*-
class Group:
    groupMembers = []
    def __init__(self, groupScope, ownerId, groupName, groupMembers=None, groupId=None):
            self.groupScope = groupScope #public, private, reserved
            self.ownerId = ownerId
            self.groupName = groupName
            if groupMembers is not None:
                self.groupMembers = groupMembers
            else:
                self.groupMembers = []
            if groupId is not None:
                self.groupId = groupId
            else:
                self.groupId = None

File: clients/iPhone/Filelocker/Models/Group.cs
using System;
using System.Collections.Generic;
namespace Filelocker
{
	public class Group
	{
		public string groupName {get; set;}
        public string groupId {get; set;}
        public List<User> groupMembers {get; set;}
		public List<string> filesSharedWithGroup {get; set;}
				
		public Group (string groupName, string groupId)
		{
			this.groupName = groupName;
			this.groupId = groupId;
			groupMembers = new List<User>();
			filesSharedWithGroup = new List<string>();
		}
	}
}